components/integrations/installed_command.jsx

```diff
@@ -60,7 +60,7 @@ export default class InstalledCommand extends React.PureComponent {
         this.props.onDelete(this.props.command);
     }
 
-    matchesFilter(command, filter) {
+    static matchesFilter(command, filter) {
```

Comment:
We don't follow this pattern in the code (at least extensively), looks weird to define a static method on this class here and import the class to use that method, maybe we can move it to utilities.

Reply:
I went with the approach that was already implemented in components/integrations/installed_incoming_webhook.jsx and components/integrations/installed_outgoing_webhook.jsx .
Each component has it's own filter logic.
I can move it from static method to an exported function if you think that will be cleaner.

Reply:
I think it will be cleaner. Thinking about this, instead of a function that filter the elements and return a list, maybe passing the nodes directly, and being sure every single node (Component instance) have the `matchesFilter` method, you can filter directly from `BackstageList`.